conf/defaults.yaml

```diff
@@ -98,6 +98,7 @@ ui.header.buffer.bytes: 4096
 ui.http.creds.plugin: org.apache.storm.security.auth.DefaultHttpCredentialsPlugin
 ui.pagination: 20
 ui.disable.http.binding: true
+ui.disable.spout.lag.monitoring: true
```

Comment:
Shouldn't this be false?

Reply:
@srdo  I would rather turn off the feature by default. I can make it `false` if you think otherwise.

Reply:
No, it's fine by me. I just wanted to be sure it wasn't a mistake.